gno.land/pkg/sdk/vm/convert_json.go

```diff
@@ -0,0 +1,38 @@
+package vm
+
+import (
+	"encoding/json"
+	"reflect"
+
+	"github.com/gnolang/gno/gnovm/pkg/gnolang"
+	gno "github.com/gnolang/gno/gnovm/pkg/gnolang"
+	"github.com/gnolang/gno/tm2/pkg/amino"
+)
+
+func UnmarshalJSON(alloc *gno.Allocator, store gno.Store, b []byte, t gno.Type) (tv gno.TypedValue, err error) {
```

Comment:
Nitpick: please don't use named returns :)

both for this and `UnmarshalNativeValueJSON`